docs/user/visual-diff.rst

```diff
@@ -12,18 +12,29 @@ and focus your review on what has changed in the current page.
 
    Example of Visual diff
 
-Enabling Visual diff
---------------------
+Using Visual diff
+-----------------
+
+Visual diff is enabled by default and it's only available on pull request builds.
+Once enabled, a new UI element appears at the top-right of the page showing a dropdown selector containing
+all the files that have changed between the base version (e.g. ``latest``) and the current pull request build.
+
+You can select any of those files from the dropdown to jump directly into that page.
+Once there, you can toggle it on/off by pressing the :guilabel:`Show diff` link from the UI element to show all the chunks and their differences highlighted with red/green background colors.
+Besides, you can jump between each of these chunks by clinking on the up/down arrows.
+
+All the available configuration for the visual diff addon can be found under :guilabel:`Settings > Addons > Visual diff` in the :term:`dashboard`.
 
-Visual diff is only enabled on pull request builds,
-and can be toggled on and off with the ``d`` hotkey.
 
 Troubleshooting Visual diff
 ---------------------------
 
-Visual diff only works if there are changes on the page,
+Visual diff only works when we detect changes on the page,
 so ensure you are on a page that has changed in the current pull request.
 
-There are also some known issues that currently don't display properly:
+There are also some known issues that currently don't display properly.
+We are working to improve the UX, but so far we've found the following issues:
 
 * **Tables** are shown to have changes when they may not have changed. This is due to do subtly in how HTML tables are rendered, and will be fixed in a future version.
+* **Invisible changes** sometimes are marked as diff due than the underlying HTML changed, but there is no visual change. This could happen if the URL of a link changed, for example.
```

Comment:
```suggestion
* **Invisible changes** sometimes are marked as diff due than the underlying HTML changing, but there is no visual change. This could happen if the URL of a link changed, for example.
```